tests/from_msa/test_cluster_sequences.py

```diff
@@ -383,11 +383,11 @@ def test_GivenAllSequencesSmallEditDist_ReturnsNoClustering(self):
             self.assertEqual(expected_clustering, result)
 
     def test_GivenManyVeryDifferentSequences_EachSeqInOwnCluster(self):
-        # all 1024 distinct DNA 5-mers.
-        # We want each in one cluster, but do not want to run clustering 1023 times
-        all_5mers = list(map("".join, product(standard_bases, repeat=5)))
+        # all 256 distinct DNA 5-mers.
+        # We want each in one cluster, but do not want to run clustering 255 times
+        all_5mers = list(map("".join, product(standard_bases, repeat=4)))
```

Comment:
Being picky here, but should be `all_4mers` not `all_5mers`
Also, i realise this comment `# We want each in one cluster, but do not want to run clustering 255 times` is slightly off (my bad), and should be `# We want clustering to keep looking for clusters, and stop at MAX_CLUSTERS (instead of running 255 times)`

Reply:
Do let me know if youd like me to do this (not sure how though, could I push to mbhall88:master ?)